ci/hosts

```diff
@@ -8,4 +8,12 @@ localhost ansible_connection=local
 
 [stockpile]
 # The host stockpile is running from
-localhost ansible_connection=local
+localhost ansible_connection=local stockpile_user=root stockpile_output_path=/tmp/stockpile.json
```

Comment:
Setting the vars here would seem to be redundant with group_vars/all.yml

Reply:
when I don't set stockpile_user=root stockpile_output_path=/tmp/stockpile.json in the ci/hosts file and run ```config/featureset001.yml ``` I get  http://paste.openstack.org/show/788393/

Reply:
Sorry for the lag in response. I believe you get this because the group_vars directory is relative to the yaml you are running. I.e. since there is no group_vars/all.yml file in config then it can't find any of those variables. If you copy featureset001.yml to the root of the stockpile directory then run it from there it does not through that error.

Some options we could use to work around this:
1) Assume the user (or ci) will copy that to the right dir
2) link the group_vars dir to config/group_vars
3) Manually load the group_vars file in the featureset yaml via vars_file

Thoughts?

Reply:
@dry923 Done. Thanks have pushed the changes. Could you please review